contracts/0.4.24/interfaces/ILido.sol

```diff
@@ -79,16 +153,22 @@ interface ILido {
       */
     function getWithdrawalCredentials() external view returns (bytes);
 
-
     event WithdrawalCredentialsSet(bytes32 withdrawalCredentials);
 
+    /**
+    * @dev Sets given address as the address of LidoMevTxFeeVault contract
+    * @param _mevTxFeeVault MEV and Tx Fees Vault contract address
+    */
+    function setMevTxFeeVault(address _mevTxFeeVault) external;
+
+    event LidoMevTxFeeVaultSet(address mevTxFeeVault);
 
     /**
       * @notice Ether on the ETH 2.0 side reported by the oracle
```

Comment:
need to update the docs here (should match the ones in `Lido.sol`)

Reply:
you've marked the prev conversation as resolved but the doc still doesn't reflect what we have in `Lido.sol`